components/esp_modem/test/host_test/CMakeLists.txt

```diff
@@ -4,9 +4,10 @@ include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 
 set(EXTRA_COMPONENT_DIRS    # Add esp_modem component and linux port components
         ../..
-        ../../port/linux)
+        ../../port/linux
+        "$ENV{IDF_PATH}/tools/mocks/freertos/")
```

Comment:
Why do we need freertos mocks?

Reply:
https://github.com/gabsuren/esp-protocols-1/actions/runs/7100388129/job/19326385718